src/V3File.cpp

```diff
@@ -824,11 +829,19 @@ void V3OutFormatter::putsQuoted(const string& strg) {
     putcNoTracking('"');
 }
 void V3OutFormatter::putsNoTracking(const string& strg) {
+    if (!v3Global.opt.formatCxxCode()) {
+        for (const char* cp = strg.c_str(); *cp; cp++) { putcOutput(*cp); }
```

Comment:
```suggestion
        for (const char* cp = strg.c_str(); *cp; cp++) putcOutput(*cp);
```
Again puts should work

Reply:
Fixed.